buf/registry/module/v1beta1/label_service.proto

```diff
@@ -175,6 +175,9 @@ message ListLabelHistoryRequest {
     (buf.validate.field).string.uuid = true,
     (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
   ];
+  // Only list Commits where the Digest has changed from the previous Commit in the
+  // history of this Label.
+  bool only_commits_with_changed_digests = 8;
```

Comment:
Which Digest type? The one requested? 

B4 digests include config files, so I suspect you mean B5, but one may think this is the Digest type from the request.

Reply:
Not sure, let's discuss in sync today. The intent here is to only show commits where observable changes happened (aka there is a diff, aka something material to the module changed), and we are using the digest as a way to express this. Either way, I would want make sure it is not possible to set this to true and return a history where two sequential commits have the same requested digest because that would be confusing.

Reply:
Decided on B5 comparison. For v1 this is the only digest anyway.